wiki/Gameplay/Score/Ranked_score/en.md

```diff
@@ -1,9 +1,14 @@
----
-stub: true
----
-
 # Ranked score
 
 *For other uses, see [Score (disambiguation)](/wiki/Disambiguation/Score).*
 
-The **ranked score** is the total sum of the best scores for all ranked difficulties played online.
+The **ranked score** is the total sum of the best scores for all ranked, loved, and approved [difficulties](/wiki/Beatmap/Difficulty) played online.
+
+- History  (6 October 2007 – 24 July 2012)
```

Comment:
would rather just have a clean heading like this. the timeline is conveyed by the below paragraph well enough i think

```suggestion
## History
```